src/ain/node/node.controller.ts

```diff
@@ -13,11 +13,52 @@ import { NodeService, NodeType } from './node.service';
 export class NodeController {
   constructor(private readonly nodeService: NodeService) {}
 
-  @Post(':node/:mode/rpc')
+  @Post(':node/connected/rpc')
```

Comment:
Can we omit the 'connected' in the path (also on lines 27 and 41)? Would be nicer without ;-)